cmd/baton-ldap/main.go

```diff
@@ -19,6 +19,8 @@ func main() {
 	ctx := context.Background()
 
 	cfg := &config{}
+	cfg.ClientID = "baton-ldap"
```

Comment:
Should we be hard setting the clientID to this?

Reply:
oops we should not. I didn't mean to commit that